INFO[2023-04-13 17:00:58,375]: Unable to initialize backend 'cuda': module 'jaxlib.xla_extension' has no attribute 'GpuAllocatorConfig'
INFO[2023-04-13 17:00:58,376]: Unable to initialize backend 'rocm': module 'jaxlib.xla_extension' has no attribute 'GpuAllocatorConfig'
INFO[2023-04-13 17:00:58,377]: Unable to initialize backend 'tpu': INVALID_ARGUMENT: TpuPlatform is not available.
INFO[2023-04-13 17:00:58,377]: Unable to initialize backend 'plugin': xla_extension has no attributes named get_plugin_device_client. Compile TensorFlow with //tensorflow/compiler/xla/python:enable_plugin_device set to true (defaults to false) to enable this.
WARNING[2023-04-13 17:00:58,377]: No GPU/TPU found, falling back to CPU. (Set TF_CPP_MIN_LOG_LEVEL=0 and rerun for more info.)
SIMBA SAFETY LOAD DEBUG:	 get_args
43 i
bad login of ./SIMBA_jobstatus.dat attempt 1 in get_args
good login of ./SIMBA_jobstatus.dat in get_args
Traceback (most recent call last):
  File "/data/uqhmcdou/HM_Lag_Recovery/LineFit01.py", line 126, in <module>
    main()
  File "/data/uqhmcdou/HM_Lag_Recovery/LineFit01.py", line 50, in main
    job_args = SIMBA.get_args(args.i, table_url=args.table)
  File "/data/uqhmcdou/HM_Lag_Recovery/SIMBA.py", line 92, in wrapped_func
    return( func(*args, **kwargs) )
  File "/data/uqhmcdou/HM_Lag_Recovery/SIMBA.py", line 187, in get_args
    table = pd.read_csv(table_url, sep='\t', index_col=0)
  File "/home/uqhmcdou/.conda/envs/lag_conda/lib/python3.10/site-packages/pandas/util/_decorators.py", line 211, in wrapper
    return func(*args, **kwargs)
  File "/home/uqhmcdou/.conda/envs/lag_conda/lib/python3.10/site-packages/pandas/util/_decorators.py", line 331, in wrapper
    return func(*args, **kwargs)
  File "/home/uqhmcdou/.conda/envs/lag_conda/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 950, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/uqhmcdou/.conda/envs/lag_conda/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 605, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/uqhmcdou/.conda/envs/lag_conda/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1442, in __init__
    self._engine = self._make_engine(f, self.engine)
  File "/home/uqhmcdou/.conda/envs/lag_conda/lib/python3.10/site-packages/pandas/io/parsers/readers.py", line 1735, in _make_engine
    self.handles = get_handle(
  File "/home/uqhmcdou/.conda/envs/lag_conda/lib/python3.10/site-packages/pandas/io/common.py", line 856, in get_handle
    handle = open(
FileNotFoundError: [Errno 2] No such file or directory: './SIMBA_jobstatus.dat'
